lib/cdo/rack/deflater.rb

```diff
@@ -109,7 +109,7 @@ def close
     class DeflateStream
       # TODO: The `||=` is used below instead of `=` so as to avoid a constant redefinition warning.
       # Fix this, so that direct assignment can be used.
```

Comment:
I went on a journey through these comments and found [this comment thread](https://github.com/code-dot-org/code-dot-org/pull/19545#discussion_r155679139) that asked the same questions you're asking in this PR.

I'll be honest that I don't know if this change would be safe or not, but it sounds like no one else was sure either 🤷‍♀️ I would ask that you remove these comments though if you go with the `=`

Reply:
Oh, thanks! That's great context; I checked through the drone logs, and although I do see a number of instances of `warning: previous definition of SOME_CONSTANT was here`, I don't see any for `CDO` or `DEFLATE_ARGS`. Could be we've changed other things in such a way that the constant redefinition warning they reference no longer shows up, or it could be that I need to do something else to see it.

It does sound to me from that discussion that this change should be _safe_, it just might also be noisy. I'll keep an eye out!

And good catch the on comment; that's what I get for relying on the auto-corrector :upside_down_face: